### DISCUSSION TITLE
Locating an object in a long clip

### DISCUSSION THREAD

**Discussion by ronisaacson** [blakeblackshear/frigate#6322] [Ask A Question] [answered]
Last night one of my cameras recorded a long clip (about 20 minutes) because it's windy and the trees are in constant motion. Somewhere in the clip it correctly detected an object and showed the thumbnail. But I wanted to see the context and I had to watch the whole clip (on 4x playback) until I found it.

Within a clip, does frigate store a timestamp of when an object is first detected so you can easily jump to that spot?

**Comment by NickM-27** [comment, marked as answer]
Currently no there's no info for this, but in the next version you'll be able to seek directly to specific moments in the video that objects were detected, entered zones, etc.
